packages/spaces/test/commands/wait_test.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 /* globals describe beforeEach it */
 
+let sinon = require('sinon')
```

Comment:
sinon should also be listed as a dependency

Reply:
@jdxcode good catch, thanks!